### ISSUE TITLE
Blocked on "Jobs" view when there are no jobs

### ISSUE THREAD

**Issue by mehdid** [edf-hpc/slurm-web#143]
If a user happens to pick a cluster without jobs listed, he won't be able to switch to another cluster before reloading the dashboard or changing for another view before choosing a different cluster.

**Comment by rezib** [comment]
Duplicate of #136.
